rpc/cloudagent/cloud/etcdserver/moc_cloudagent_etcdserver.proto

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache v2.0 license.
+
+syntax = "proto3";
+option go_package = "github.com/microsoft/moc/rpc/cloudagent/cloud";
+package moc.cloudagent.etcdserver;
+
+import "google/protobuf/wrappers.proto";
+import "moc_common_common.proto";
+
+message EtcdServerRequest {
```

Comment:
About the hierarchy, I was asking to put etcdserver under etcdcluster, and use etcdcluster as its parent.

The equivalent is Secrets under KeyVault.

So the cli would look like 

mocctl cloud etcdcluster list
modctl cloud ectdcluster server list

Reply:
I see, will amend.

Reply:
Ok, switched to the model similar to keyvault/secret. I can see that keyvault has secrets, yet I do not see any code updating that. Is that on purpose? @madhanrm